canister/src/api/send_transaction.rs

```diff
@@ -5,7 +5,7 @@ use crate::{
 use bitcoin::{consensus::Decodable, Transaction};
 use ic_btc_types::SendTransactionRequest;
 
-pub async fn send_transaction(request: SendTransactionRequest) {
+pub async fn send_transaction(request: SendTransactionRequest) -> Result<(), String> {
```

Comment:
When possible, we should not return errors as strings, as these are hard to parse, and the caller can't easily distinguish between the various cases. I know in this case there's only one error, so it doesn't matter too much, but it's good to stay principled and consistent. I'd define an enum for the error, and that translate that error into a string in the calling function.

```
enum SendTransactionError {
  CannotDecode
}
```

Reply:
Done.